.formatter.exs

```diff
@@ -1,5 +1,5 @@
 [
-  import_deps: [:ecto, :phoenix],
+  import_deps: [:ecto, :phoenix, :md],
```

Comment:
To allow DSL from `Md` without parentheses, like `magnet "@", %{transform: Handler}`.